config/initializers/content_security_policy.rb

```diff
@@ -0,0 +1,49 @@
+require 'feature_management'
+
+if FeatureManagement.rails_csp_tooling_enabled?
+  Rails.application.config.content_security_policy do |policy|
+    connect_src = ["'self'", '*.nr-data.net', '*.google-analytics.com', 'us.acas.acuant.net']
+    connect_src << %w[ws://localhost:3035 http://localhost:3035] if Rails.env.development?
```

Comment:
This line can be removed in favor of the ones in the `if ENV['WEBPACK_PORT']` below.

```suggestion
```

Reply:
nice, good tip!